package.json

```diff
@@ -97,7 +97,7 @@
     "@types/lodash.isequal": "^4.5.2",
     "@types/lodash.set": "^4.3.6",
     "@types/node": "*",
-    "@types/puppeteer": "1.19.x",
```

Comment:
was this necessary for anything here? I'd feel a little safer reverting if not. The fact that we specifically pinned to `1.19.x` makes me think we had a reason to do so in the past (or at least I hope we did 😆)